### DISCUSSION TITLE
Laravel Backpack Page Designer

### DISCUSSION THREAD

**Discussion by lucasheer** [Laravel-Backpack/community-forum#707] [Show and Tell]
I discovered Laravel Backpack yesterday, and am loving this platform!! I integrated a project: 'HansSchouten/Laravel-Pagebuilder' into Laravel Backpack to start transferring some of my websites over to this platform. This project uses a page editor called GrapeJS.

https://drive.google.com/file/d/1ViXekeqMDSzWIhWmOKJ-boRaLZfFBc-w/view?usp=sharing

I plan to make this a plugin for BackPack 👍

**Comment by promatik** [comment]
WOW! @lucasheer, that is absolutely awesome!

I didn't know about that `Laravel-Pagebuilder`, but it looks super funcional 👌
This will indeed be an helpful package for the Backpack community.

I'm eager to see it in action, I'll be happy to review your package 🙌

**Comment by tabacitu** [comment]
That sounds awesome @lucasheer ! If you do create a package for it, please let us know - happy to test, review, promote it 🎉

**Comment by dottwatson** [comment]
no updates? I'm interested ini it :-)
